presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -937,6 +937,30 @@ public void testCastNullToColumnTypes()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateEmptyPartition()
+    {
+        String tableName = "empty_partition_table";
+        assertUpdate(format("" +
+                "CREATE TABLE %s " +
+                "WITH ( " +
+                " FORMAT = 'ORC', " +
```

Comment:
Nit: lowercase the name `format` and use consistent indentation with the other property (two spaces)